Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/Issue12153.cs

```diff
@@ -0,0 +1,43 @@
+﻿using Xamarin.Forms.CustomAttributes;
+using Xamarin.Forms.Internals;
+
+#if UITEST
+using Xamarin.Forms.Core.UITests;
+using Xamarin.UITest;
+using NUnit.Framework;
+#endif
+
+namespace Xamarin.Forms.Controls.Issues
+{
+#if UITEST
+	[Category(UITestCategories.ManualReview)]
+#endif
+	[Preserve(AllMembers = true)]
+	[Issue(IssueTracker.Github, 12153, "Setting FontFamily to pre-installed fonts on UWP crashes", PlatformAffected.UWP)]
+	public class Issue12153 : TestContentPage // or TestMasterDetailPage, etc ...
```

Comment:
Can you add a UITest? If not so I can add one

Reply:
I have no idea how to do that for something like this